AirCasting/CreateSessionViews/PowerABView.swift

```diff
@@ -24,8 +24,7 @@ struct PowerABView: View {
             Image("2-power")
             VStack(alignment: .leading, spacing: 13) {
```

Comment:
Something's wrong with alignment, title displays in the middle, should be leading :)